AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -1681,18 +1681,6 @@ - (void)nodeDidRelayout:(ASCellNode *)node sizeChanged:(BOOL)sizeChanged
   _nextLayoutInvalidationStyle = invalidationStyle;
 }
 
-#pragma mark - Memory Management
-
-- (void)clearContents
-{
-  [_rangeController clearContents];
```

Comment:
@garrettmoon I thought these methods could get called by exiting the window -- I think we probably still need to support this for 2.0 compatibility, unless apps don't compile when they try to create an ASCV directly?  Is that direct-CV-init just deprecated, or gone?